django/db/models/query.py

```diff
@@ -165,9 +165,9 @@ def __init__(self, model=None, query=None, using=None, hints=None):
         self._result_cache = None
         self._sticky_filter = False
         self._for_write = False
-        self._prefetch_related_lookups = []
+        self._prefetch_related_lookups = ()
         self._prefetch_done = False
-        self._known_related_objects = {}  # {rel_field, {pk: rel_obj}}
+        self._known_related_objects = {}  # {rel_field: {pk: rel_obj}}
```

Comment:
N.B. I fixed this whilst I was reading it, it doesn't seem worth extracting into a second patch despite it not being entirely related